slave-setup/README.md

```diff
@@ -0,0 +1,24 @@
+#Setting up slaves using Slave Setup Plugin
+
+##Pre-requisites to setup a slave 
+
+Once you have obtained a VM;
+ 1. Install latest jdk
+ 2. Create a directory /build/jenkins-home at root level
+
+The following should be available in /build/worker_installer/slaveSetupFile location in the Jenkins master ;
+ 1. jdk-7u51-linux-x64.tar.gz
+ 2. jdk-8u144-linux-x64.tar.gz
+ 3. apache-maven-2.2.1-bin.tar.gz
+ 4. apache-maven-3.0.5-bin.tar.gz
+ 5. apache-maven-3.1.1-bin.zip
+ 6. apache-maven-3.2.2-bin.tar.gz
+ 7. apache-maven-3.3.9-bin.zip
+ 8. UnlimitedJCEPolicyJDK7.zip
+ 9. jce_policy-8.zip
+ 10. node-v6.10.0-linux-x64.tar.xz
```

Comment:
I guess we do not need to manually copy `node` archive now. \o/

We do need to automate the apache-maven-* download as well. 

I just found out that we can automate JDK downloading as well. Read here - https://stackoverflow.com/a/10959815/388714

So, only thing pending then is downloading JCE policies. But these are also coming from Oracle. Therefore, the solution suggested in the above SO answer is applicable here as well.